src/filedialog.cpp

```diff
@@ -965,10 +965,13 @@ void FileDialog::FileDialogFilter::update() {
         }
         nameFilter = nameFilter.mid(left, right - left);
     }
-    // parse the "*.ext1 *.ext2 *.ext3 ..." list into QRegExp objects
+    // parse the "*.ext1 *.ext2 *.ext3 ..." list into QRegularExpression objects
     auto globs = nameFilter.simplified().split(' ');
     for(const auto& glob: globs) {
-        patterns_.emplace_back(QRegExp(glob, Qt::CaseInsensitive, QRegExp::Wildcard));
+        // NOTE: Use QRegularExpression::anchoredPattern() later, when Qt 5.12 is required.
```

Comment:
For Qt 5.12, it's OK to use conditional compilation here?
`#if QT_VERSION >= ....`

Reply:
LGTM.

Reply:
> For Qt 5.12, it's OK to use conditional compilation here?

IMHO, there was no need to that because it does exactly the same thing. However, at some time, Qt 5.12 will be the minimum dependency.